### PR TITLE
feat: added badge

### PR DESCRIPTION
**Pull request by anamkaa**
Added variations of badge components
- badge on icon
- badge with text on icon
- badge on avatar
- badge on button 
- badge with text on button
- text badges of different colors
- text badges-outline of different colors
- text pill badges of different colors
- text pill-outline of different colors

Screenshot (229)

### CODE REVIEW

**Review by shubhamkumar648 — COMMENTED** [review]
look good! go ahead

File: stylesheets/components/badge.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  position: relative;
+  margin: 1rem;
+}
+
+.bdg-i {
+  font-size: 2rem;
+}
+
+.badge-icon {
+  left: 60%;
```
Comment (vivekmehto) [inline stylesheets/components/badge.css:14]: where did you use this class? Didn't get this.

**Comment by hrshmistry** [comment]
Hello @anamkaa, badge with text on button looks fantastic, loved the variations 👍🏻

**Review by vivekmehto — COMMENTED** [review]
Everything looks good except badge icon class.

File: stylesheets/components/badge.css (hunk 1 above)
Comment (anamkaa) [reply stylesheets/components/badge.css:14]: I have used this class to position badge for my icons.

**Pull request merged**